src/kanata/linux.rs

```diff
@@ -31,15 +31,20 @@ impl Kanata {
         Kanata::set_repeat_rate(&k.defcfg_items)?;
         drop(k);
 
+        let scroll_wheel_mapped = MAPPED_KEYS.lock().contains(&OsCode::MouseWheelUp)
```

Comment:
This is quite inefficient. This could be computed once per live-reload. It could also be done by locking only once instead of locking/unlocking 4 times.

I also think it's not necessary. I think it would be better to translate a scroll event into its corresponding key and check `MAPPED_KEYS` against that.

Reply:
> This could be computed once per live-reload

Oh, but currently it's not re-computed after live-reload at all... So another thing to fix.

> I also think it's not necessary. I think it would be better to translate a scroll event into its corresponding key and check `MAPPED_KEYS` against that.

I'm not sure what you are referring to as "not necessary". Can you clarify?